services/src/test/java/org/fao/geonet/api/records/extent/MetadataExtentApiTest.java

```diff
@@ -124,23 +132,34 @@ public void getOneRecordExtentAsImage() throws Exception {
             .andReturn().getResponse().getContentAsByteArray();
 
         saveImageToDiskIfConfiguredToDoSo(reponseBuffer, name.getMethodName());
-        assertEquals("b02baec6d92832ecd5653db78093a427", DigestUtils.md5DigestAsHex(reponseBuffer));
+        assertImage(
+            REFERENCE+"getOneRecordExtentAsImage.png",
+            reponseBuffer,
+            550);
     }
 
-    @Test
+    @Ignore
```

Comment:
cf. https://github.com/geonetwork/core-geonetwork/blob/0eba538258926e682fb66709de8776e515df968a/services/src/test/java/org/fao/geonet/api/records/extent/MetadataExtentApiTest.java#L371

last modification date is "2015, Calendar.OCTOBER, 21, 07, 28, 0"

response should be not modified ?

version available at browser side date is "Wed, 21 Oct 2015 07:29:00 UTC"

Reply:
If the test is showing a new bug it can be addressed in a separate PR. I do wonder if the failure in this case is timezone sensitive ...

Reply:
If service return "not modified" it is not a bug. Otherwise, the test should be deleted.